docs/spec/src/integration/spec/6-secure-integration.md

```diff
@@ -1,29 +1,44 @@
 # Secure Integration
 
-This page is meant to be read by eigenda and rollup developers who are writing a secure integration. To understand what a secure integration is at a high level, visit our [secure integration overview](https://docs.eigenda.xyz/integrations-guides/rollup-guides/integrations-overview) docs instead.
+This page is meant to be read by eigenda and rollup developers who are writing a secure integration and need to understand the details. For users who just want a high-level understanding of what a secure integration is, please visit our [secure integration overview](https://docs.eigenda.xyz/integrations-guides/rollup-guides/integrations-overview) page instead.
 
-## Cert+Blob+Timing Validation
+## Validity Conditions
 
-Blob and Cert verification is done for both the write (sequencer) and read (validator) paths. Given this duplication in the lifecycle, and given its importance, it deserves its own section.
+EigenDA is a service that assures the availability and integrity of payloads posted to it for 14 days.
+When deriving a rollup chain by running its derivation pipeline, only EigenDA certs that satisfy 3 validity conditions are considered valid and used:
+1. RBN Recency Validation - ensure that the cert's reference block number (RBN) is not too old with respect to the L1 block at which the cert was included in the rollup's batcher-inbox. This ensures that the blob on EigenDA has sufficient availability time left (out of the 14 day period) in order to be downloadable if needed during a rollup fraud proof window.
+2. Cert Validation - ensures sufficient operator stake has signed to make the blob available, for all specified quorums. The stake is obtained onchain at a given reference block number (RBN) specified inside the cert.
+3. Blob Validation - ensures that the blob used is consistent with the KZG commitment inside the Cert.
 
-The validation process is:
+### 1. RBN Recency Validation
 
-1. Validate the Cert (against state on ethereum)
-2. Validate the Blob (against the Cert)
+First, we assume that every rollup stack has a mechanism to force blobs to be included in the rollup’s batcher-inbox in a timely matter, such that the cert's blob is still available on EigenDA for long enough to cover the fraud proof duration (typically 7 days), or long enough to be downloadable by zk rollup participants.
 
-### Cert Validation
+> For example, optimism has a [sequencerWindow](https://docs.optimism.io/stack/rollup/derivation-pipeline#sequencer-window) which forces batches to land onchain in a timely fashion (12h). This indirectly also forces EigenDA certs to land onchain in a timely fashion. Therefore, it is safe to assume that the certs land onchain < SequencerWindowSize after having been signed.
+>
+> OP stack rollups thus need to make sure that `SequencerWindow` << 14 days.
```

Comment:
Why are we setting the `RecencyWindow=SequencerWindow`? iiuc there's a liveness x safety tradeoff for the value assumed for the `RecencyWindow` where increasing the window increases liveness while decreasing safety. Ie, for optimistic rollups you'd need the data available for 14 days for ensuring successful fraud proofs by an honest party. This creates an attack zone boundary starting at `MaxChallengeWindow + MaxResolveWindow - RecencyWindow` and ending at `MaxChallengeWindow + MaxResolveWindow`. Curious how 12 hours is the optimal expression given we could put it down to e.g, 30-minutes to one hour while still ensuring liveness at the benefit of greater safety

Reply:
TLDR is just that I don't think it matters that much, and this way we don't need to introduce a new variable in the rollup.json config. So we keep our fork minimal. Also RecencyWindow and SequencerWindow are related, I think RecencyWindow needs to be <= SequencerWindow (since anyways if you wait longer the batches contained inside the blob will get discarded by the batch_queue stage as bowen pointed out in https://github.com/Layr-Labs/eigenda/pull/1533#discussion_r2078526064